### PR TITLE
create HTAN DCC buckets for TMA-TNP, MSK, and Vanderbilt

### PR DESCRIPTION
**Pull request by clarisse-lau**
- create HTAN DCC buckets for TMA-TNP and MSK
- create (non-lambda linked) HTAN DCC bucket for Vanderbilt

### CODE REVIEW

File: config/prod/htan-dcc-msk.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html) that will
 # work with the s3-synapse-sync lambda: https://github.com/Sage-Bionetworks/s3-synapse-sync
 template_path: "remote/s3-synapse-sync-bucket.j2"
-stack_name: "htan-dcc-bu"
+stack_name: "htan-dcc-msk"
```
Comment (zaro0508) [inline config/prod/htan-dcc-msk.yaml:4]: you cannot rename to delete.  deletion must be a separate PR

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: config/prod/htan-dcc-msk.yaml (hunk 1 above)
Comment (clarisse-lau) [reply config/prod/htan-dcc-msk.yaml:4]: thanks, updated.

File: config/prod/s3-synapse-sync.yaml [hunk 2]
```diff
@@ -13,8 +13,9 @@ parameters:
   BucketVariables: >-
     {
       "htan-dcc-center-a":{"SynapseProjectId":"syn22343754"},
-      "htan-dcc-bu":{"SynapseProjectId":"syn22124336"},
-      "htan-dcc-ohsu":{"SynapseProjectId":"syn22093319"}
+      "htan-dcc-ohsu":{"SynapseProjectId":"syn22093319"},
```
Comment (zaro0508) [inline config/prod/s3-synapse-sync.yaml:16]: if you're intention is to delete `htan-dcc-bu` bucket then please delete the htan-dcc-bu.yaml file and include it in this PR.

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: config/prod/s3-synapse-sync.yaml (hunk 2 above)
Comment (clarisse-lau) [reply config/prod/s3-synapse-sync.yaml:16]: when I try to delete the file, it's treated as renaming to the new bucket. how can I avoid this?

Comment (clarisse-lau) [reply config/prod/s3-synapse-sync.yaml:16]: @zaro0508 should I just include the bucket deletion in a separate PR?

Comment (zaro0508) [reply config/prod/s3-synapse-sync.yaml:16]: yes, just do in a separate PR if you can't figure out the GIT thing.

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: config/prod/s3-synapse-sync.yaml [hunk 3]
```diff
@@ -13,8 +13,9 @@ parameters:
...
+      "htan-dcc-tma-tnp":{"SynapseProjectId":"syn22041595"},
+      "htan-dcc-msk":{"SynapseProjectId":"syn23448901"}
```
Comment (zaro0508) [inline config/prod/s3-synapse-sync.yaml:18]: don't you want to add vanderbilt bucket to this list?

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: config/prod/s3-synapse-sync.yaml (hunk 3 above)
Comment (clarisse-lau) [reply config/prod/s3-synapse-sync.yaml:18]: Not at the moment. vanderbilt will upload data via Synapse, and we just plan to attach this bucket to their project as an external storage location (not set up with the sync), like the CHOP bucket Xengie provisioned last week.

**Review by zaro0508 — APPROVED** [review]
